### PR TITLE
[MIG] product_supplierinfo_for_customer_sale: Migration to 13.0

### PR DESCRIPTION
**Pull request by PabloPaezSheridan**

### CODE REVIEW

File: product_supplierinfo_for_customer_sale/__manifest__.py [hunk 1]
```diff
@@ -3,7 +3,7 @@
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
 {
     "name": "Product Supplierinfo for Customer Sale",
-    "version": "12.0.1.1.0",
+    "version": "13.0.1.1.0",
```
Comment (jjscarafia) [inline product_supplierinfo_for_customer_sale/__manifest__.py]: small detail, but it should be 13.0.1.0.0 (the migration script didn't correct it automatically?)

File: product_supplierinfo_for_customer_sale/tests/test_product_supplierinfo_for_customer_sale.py [hunk 2]
```diff
@@ -43,19 +43,14 @@ def setUp(self):
...
-                "email": "example@yourcompany.com",
-                "customer": True,
-                "phone": 123456,
-            }
+            {"name": name, "email": "example@yourcompany.com", "phone": 123456}
         )
 
     def _create_partnerinfo(
-        self, supplierinfo_type, partner, product, empty_variant=False
+        self, supplierinfo_type, customer, product, empty_variant=False
```
Comment (jjscarafia) [inline product_supplierinfo_for_customer_sale/tests/test_product_supplierinfo_for_customer_sale.py]: why are we renaming this argument?

File: product_supplierinfo_for_customer_sale/tests/test_product_supplierinfo_for_customer_sale.py [hunk 3]
```diff
@@ -43,19 +43,14 @@ def setUp(self):
...
     ):
         vals = {
-            "name": partner.id,
+            "name": self.customer.id,
```
Comment (jjscarafia) [inline product_supplierinfo_for_customer_sale/tests/test_product_supplierinfo_for_customer_sale.py]: why not using customer.id? (actually if should be partner.id if the argument is not renamed)

**Review by jjscarafia — CHANGES_REQUESTED** [review]

File: product_supplierinfo_for_customer_sale/__manifest__.py (hunk 1 above)
Comment (PabloPaezSheridan) [reply product_supplierinfo_for_customer_sale/__manifest__.py]: No, the changes were only suggeted but not maked by the pre-commit feature. Okey understood, then i commit that

File: product_supplierinfo_for_customer_sale/tests/test_product_supplierinfo_for_customer_sale.py (hunk 3 above)
Comment (PabloPaezSheridan) [reply product_supplierinfo_for_customer_sale/tests/test_product_supplierinfo_for_customer_sale.py]: was only for better understanding across the debugging, nothing functional at all, i'll make the changes you suggest

File: product_supplierinfo_for_customer_sale/tests/test_product_supplierinfo_for_customer_sale.py (hunk 2 above)
Comment (PabloPaezSheridan) [reply product_supplierinfo_for_customer_sale/tests/test_product_supplierinfo_for_customer_sale.py]: partner_id is a mandatory many2one field in the sale.order model

**Review by jjscarafia — APPROVED** [review]
LGTM

**Review by danimv5 — APPROVED** [review]
Functional review. LGTM!

**Comment by rousseldenis** [comment]
@PabloPaezSheridan Could you squash your commits ?

**Comment by PabloPaezSheridan** [comment]
> @PabloPaezSheridan Could you squash your commits ?

This all done

**Comment by LoisRForgeFlow** [comment]
@PabloPaezSheridan I did not see this pr and created a new migration here: https://github.com/OCA/sale-workflow/pull/1667. However it also include an improvement to use a centralized method to get the customer code (check https://github.com/OCA/product-attribute/pull/891).

do you think we can supersede this pr with https://github.com/OCA/sale-workflow/pull/1667?

Thanks!

**Comment by rafaelbn** [comment]
Already merged in https://github.com/OCA/sale-workflow/pull/1667#issuecomment-891609528

**Pull request closed without merging**